index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header-logo-wrapper">
+        <img class="header-logo" src="/img/logo.svg" alt="Grand Players" />
```

Comment:
> すみません。絶対パスを使っている所を自分には見つけることが出来ませんでした。 /img/logo.svgのところが絶対パスになっているのでしょうか？

そうです。相対パスで表すと以下のようになります。
```suggestion
        <img class="header-logo" src="./img/logo.svg" alt="Grand Players" />
```

Reply:
なるほど！！
理解しました！！！
修正しました！